cmd/yggd/client.go

```diff
@@ -299,6 +299,13 @@ func (c *Client) ReceiveControlMessage(msg *yggdrasil.Control) error {
 			if err := c.transporter.Connect(); err != nil {
 				return fmt.Errorf("cannot reconnect to broker: %w", err)
 			}
+		case yggdrasil.CommandNameCancel:
+			log.Info("cancelling message...")
+			// Dispatch to apropriated worker.
+			if err := c.dispatcher.CancelMessage(cmd, msg.MessageID); err != nil {
```

Comment:
I needed a way to pass the MessageID of the control message (The new message). The id of the message to be cancel is inside the arguments of the control message.

The Cancel method expects:

`directive` ->  worker name
`id` -> id of the message (same as the newly created)
`messageID` -> to be canceled

This will be more clear with the new arguments that you proposed https://github.com/RedHatInsights/yggdrasil/pull/122/files#r1287363906